chainweb.cabal

```diff
@@ -61,6 +61,7 @@ common debugging-flags
 common warning-flags
     ghc-options:
         -Wall
+        -Werror
```

Comment:
Should we also add this to the other build sections (test suite, cw-tool, etc)?

Reply:
Already is! That's what `common` is for.

Reply:
Oh, right.  My bad.